content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -439,6 +539,9 @@ email, activate the :guilabel:`E-mail` checkbox.
 .. image:: brazil/correction-letter.png
    :alt: Correction letter reason in Odoo.
 
+.. important::
+   This electronic document is only available for NF-e.
```

Comment:
Specify which document exactly (for example, "Correction letters are only available for electronic invoices for goods (NF-e)."